test/NuGet.Core.Tests/NuGet.Protocol.Tests/ServiceTypesTests.cs

```diff
@@ -20,5 +21,20 @@ public void RepositorySignatures_AllResourcesAreVersioned()
                 Assert.True(pattern.IsMatch(repositorySignaturesResource), $"RepositorySignatures resource '{repositorySignaturesResource}' is not versioned with an expected format.  Verify that {nameof(RepositorySignatureResourceProvider)} can handle any new formats.");
             }
         }
+
+        [Fact]
+        public void RegistrationsBaseUrls_Returns_All_Versions_In_Desc_Order()
+        {
+            string[] expected =
+            {
+                "RegistrationsBaseUrl/Versioned",
+                "RegistrationsBaseUrl/3.6.0",
+                "RegistrationsBaseUrl/3.4.0",
+                "RegistrationsBaseUrl/3.0.0-rc",
+                "RegistrationsBaseUrl/3.0.0-beta",
+                "RegistrationsBaseUrl"
```

Comment:
Is it correct ordering assumption? 
Does non-versioned index need to be the last one to try?

Reply:
Team Review: Yes